hack/components/bump-linux-bridge.sh

```diff
@@ -22,7 +22,7 @@ echo 'Get linux-bridge tag'
 LINUX_BRIDGE_TAG=$(git-utils::get_component_tag ${LINUX_BRIDGE_PATH})
 
 echo 'Build container image with linux-bridge binaries'
-LINUX_BRIDGE_TAR_CONTAINER_DIR=/usr/src/github.com/containernetworking/plugins/bin
```

Comment:
note that this change will make the PR fail until @phoracek runs `PUSH_IMAGES=true make bump-linux-bridge`

Reply:
I don't follow, why do we need to change this?

Reply:
we don't really :) removed